README.md

```diff
@@ -113,11 +113,11 @@ $ make qemu kvm=no
 
 #### QEMU with KVM
 
-To use QEMU with KVM (kernel-based virtual Machine), which is faster than without KVM, you need a CPU with Intel® Virtualization Technology (Intel® VT) or AMD Virtualization™ (AMD-V™) support. Most systems have this disabled by default, so you may need to reboot, go into the BIOS, and enable it.
+QEMU is faster when used with KVM(kernel-based virtual Machine) enabled. To use QEMU with KVM, you need a CPU with Intel® Virtualization Technology (Intel® VT), or AMD Virtualization™ (AMD-V™) support. This feature can be enabled in BIOS settings (most systems have this feature disabled). 
 
 ### <a name="manual-setup"> Manual Setup </a>
 
-To manually clone, build and run Redox using a Unix-based host, run the following commands (with exceptions, be sure to read the comments):
+To manually clone, build and run Redox using a Unix-based host, run the following commands. If you recieve exceptions, please go through the comments):
```

Comment:
I think it is meant that you may (exception) need to adjust the commands.

Reply:
There’s a typo: `recieve` should be written as `receive`. 🤓